ql/src/java/org/apache/hadoop/hive/ql/exec/repl/ReplLoadTask.java

```diff
@@ -286,6 +277,7 @@ a database ( directory )
 
       // Populate the driver context with the scratch dir info from the repl context, so that the temp dirs will be cleaned up later
       context.getFsScratchDirs().putAll(loadContext.pathInfo.getFsScratchDirs());
+      createReplLoadCompleteAckTask();
```

Comment:
1) During incremental when there is only bootstrap task and no  events, this might not reach here, it will go back right from Line 436: return executeBootStrapLoad();
2) How does it make sure that the ack task is added after the Repl event ID update task?

Reply:
From executeBootStrapLoad the ack task will be added. We need this ack to be added at the end before we return a status.